tests/PhpUnit/JsonSchemaConstraintTest.php

```diff
@@ -53,7 +54,8 @@ public function testValidSchema()
 JSON;
         $response = json_decode($response);
 
-        self::assertTrue($this->constraint->evaluate($response, '', true), $this->constraint->evaluate($response));
```

Comment:
I saw adding `$this->validator->reset();` to `matches()` should fix the stateful thing.

Could you add that statement and restore previous assertions?